rpcs3/Emu/System.cpp

```diff
@@ -482,9 +482,9 @@ void Emulator::Init()
 						pdone = g_progr_pdone;
 
 						// Compute new progress in percents
-						const u32 total = ftotal + ptotal;
-						const u32 done = fdone + pdone;
-						const u32 new_value = double(done) * 100. / double(total ? total : 1);
+						const u32 total = ::narrow<u32>(ftotal + ptotal);
+						const u32 done = ::narrow<u32>(fdone + pdone);
```

Comment:
I don't think the choice of using u64 for ftotal, fdone ...  g_progr_ftotal,  g_progr_fdone... over u32 was really wise here.
I'll let neko review this.